src/components/table/ExportXlsx.jsx

```diff
@@ -13,9 +13,21 @@ const ExportXlsx = (props) => {
   const { t } = useTranslation();
   const { exportUrl, params, count } = props;
   const exportXlsxModalRef = useRef();
+  const [hasPermission, setHasPermission] = useState(false);
 
   const openExportXlsxModal = () => exportXlsxModalRef.current.show();
 
+  const checkPermission = () => {
+    Api.get('payment/project/')
+      .then((resp) => {
+        resp.data.forEach((project) => {
+          if (project.status === 'active' && project.is_active && project.active_subscription.price > 0) {
```

Comment:
I suggest calculate this one time on the backend and save in Redux in user data.
`DataOceanUserSerializer`

Reply:
done